### PR TITLE
Armani Hall Quiz

### PR DESCRIPTION
**Pull request by ArmaniH**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,7 +26,11 @@ Demonstrate calling the method with an argument of "young prince".
 
 Write your code here:
 ```ruby
-# code here
+def offerRose(person)
+   puts "Would you take this rose and help out
+   an old beggar, #{person}?"
```
Comment (jshawl) [inline README.md:31]: this is perfect! just keep this string on one line

File: README.md [hunk 2]
```diff
@@ -49,7 +53,8 @@ add her to the list of guests in the castle.
 
 Write your code here:
 ```ruby
-# code here
+
+guests: [Belle] = residents.delete [Belle]
```
Comment (jshawl) [inline README.md:57]: `town[:residents].delete("Belle")`

File: README.md [hunk 3]
```diff
@@ -89,7 +96,12 @@ entities (no need to draw an ERD):
 
 Your answer:
 ```
-Replace this with your answer
+An ERD is an entity relationship diagram and is used to visually represent the relationship between database entities. They help to organize how data is stored and accessed.
+
+Genie to Lamp would be a one-to-one relationship as one Genie lives in one Lamp.
+Lamp to Person would be a one-to-many relationship as one person can own many lamps, but a lamp can only be owned by one person.
+Genie to Person would be a one-to-one relationship as one person can only have one genie at a time.
+Person to pet would be a many-to-many relationship as many people can have many pets that they share, like a family of 4 with 4 pets.
```
Comment (jshawl) [inline README.md:104]: :star2:

File: README.md [hunk 4]
```diff
@@ -100,7 +112,8 @@ SQL database. If you need an example, you can use: people and wishes
 
 Your answer:
 ```
-Replace this with your answer
+A schema organizes how database tables are constructed. Which columns are named what and contain which information.
+To use people and wishes as an example for how to represent a one-to-many relationship in a schema, the table for Person would have a few attributes like, name and age. The database would also give them a unique id. The table for Wishes would have it's own attributes like number of wish or what the wish was. We would need to include the Person's unique id as an attribute to each wish as well. So one Person can have many Wishes.
```
Comment (jshawl) [inline README.md:116]: perfect answer! We were looking for SQL here, I recommend you give it a try - happy to give feedback on this

File: README.md [hunk 5]
```diff
@@ -125,7 +138,13 @@ Write code to do the following:
 
 Write your code here:
 ```ruby
-# code here
+1. lamp1 = Lamp.create(wishes_remaining: 3)
+    genie.lamp1.create(name: "Genie")
```
Comment (jshawl) [inline README.md:142]: `genie` is undefined here

File: README.md [hunk 6]
```diff
@@ -125,7 +138,13 @@ Write code to do the following:
...
+2. lamp = Genie.find_by(genie_id:(1))
```
Comment (jshawl) [inline README.md:143]: or just `Genie.find(1)` - but this is right! well, without the parentheses

File: README.md [hunk 7]
```diff
@@ -125,7 +138,13 @@ Write code to do the following:
...
+3. lamp.update(wishes_remaining: 1)
+4. jafar = Genie.create(name: "Jafar")
+    jafar.Lamp.create(wishes_remaining: 3)
+5. genie.Lamp.destroy
```
Comment (jshawl) [inline README.md:147]: use lowercase lamp here

File: README.md [hunk 8]
```diff
@@ -140,7 +159,7 @@ would look like for such an application.
 
 Your description:
 ```
-Replace this with your answer
+RESTful routes are a combination of a method to perform an action and the path to complete it
```
Comment (jshawl) [inline README.md:162]: yes! try using the word "convention" in your answer

File: README.md [hunk 9]
```diff
@@ -148,8 +167,12 @@ The ancestors have provided an example of one route; you do the other six!
...
 GET '/warriors/:id'
   * This is the show route, which finds a warrior by ID, and displays information about that warrior.
-
-Replace this with your answer
+GET "/warriors" => "warriors#index"
+GET "/warriors/new" => "warriors#new"
+POST "/warriors" => "warriors#create"
+GET "/warriors/:id/edit" => "warriros#edit"
+PUT "/warriors/:id" => "warriors#update"
+DELETE "/warriors/:id" => "warriros#destroy"
```
Comment (jshawl) [inline README.md:175]: :+1:

File: README.md [hunk 10]
```diff
@@ -170,5 +193,10 @@ Write what an example ERB file (aka view) might look like to list all the warrio
...
 Write your code here (**NOTE: syntax highlighting doesn't work for ERB in markdown files, so ignore the colors!**):
 ```html
-<!-- code here -->
+<% @warriors.each do |warrior| %>
+<li>
+  <a href="">
+  <%= warriors.name %>
+</a>
+</li>
 ```
```
Comment (jshawl) [inline README.md:202]: :+1:

**Comment by jshawl** [comment]
:+1: Nice job on the quiz! I recommend practicing more with associating models w/ active record - http://guides.rubyonrails.org/association_basics.html let me know if you'd like to go over this together!

**Comment by ArmaniH** [comment]
Thank you for the feedback. I'm enjoying ruby and will keep working with
it!

**Pull request closed without merging**